src/gui/servernotificationhandler.cpp

```diff
@@ -81,7 +81,16 @@ void ServerNotificationHandler::slotNotificationsReceived(const QJsonDocument &j
         a._message = json.value("message").toString();
         QString s = json.value("link").toString();
         if (!s.isEmpty()) {
-            a._link = QUrl(s);
+            // if there is a real whitespace in the link, the part before the space
+            // is rendered as a link text
+            if( s.contains(QChar(' '))) {
+                const QStringList li = s.split(QChar(' '));
+                a._link = QUrl::fromEncoded(li.at(1).toLocal8Bit());
```

Comment:
ToLatin1 or toUtf8.

toLocal8bit is for path on the filesystem, or things that we want to expose in stderr

There can only be one space?